manifests/prometheus/alerts.d/diego_cell_rep_memory_capacity.yml

```diff
@@ -38,16 +38,14 @@
       - record: diego_cells_required
         expr: ceil(1.5 * diego_cell_capacity_used_pct:avg5m)
 
-      - alert: DiegoCellRepMemoryCapacity
+      - alert: DiegoCellRepsReachingTotalMemoryCapacity
         expr:  rep_memory_capacity_pct:avg5m < 35
         for: 2h
         labels:
           severity: warning
         annotations:
-          summary: Rep low free memory capacity
+          summary: Reps low free memory capacity
           bosh_job_name: "{{ $labels.bosh_job_name }}"
           description: >
-            Low free memory {{ $value | printf "%.0f" }}% for the advertised rep memory capacity
-            in the last 2 hours on average.
-            Review if we need to scale...
+            Low free memory {{ $value | printf "%.0f" }}% for the total advertised rep memory capacity in the last 2 hours on average - if we lost an AZ we may not be able to find space for all the app instances we're currently running. Review if we need to scale.
```

Comment:
This doesn't read quite right. Suggest, perhaps, something like

> Reps are reporting low free memory on average (x%) over the last 2 hours. If we lost an AZ now, we may not have space to redistribute all the apps running in the lost AZ. Review if we need to scale the cells.

Reply:
> Reps are reporting low free memory on average (x%) over the last 2 hours

The _whole point_ of rewriting this was to avoid the ambiguity that is inherent in this. Is it individual reps that are reporting low free memory? What is "low free memory"? Is that low amounts of memory free on the whole bosh instance? _Actual_ memory? _Committed_ memory? Is this just because an app instance has started using more memory than was expected? Is the "on average over the last 2 hours" taken per-cell?